Formula/freetype.rb

```diff
@@ -34,6 +34,10 @@ def install
     system "./configure", "--prefix=#{prefix}", "--without-harfbuzz"
     system "make"
     system "make", "install"
+
+    %W[#{bin}/freetype-config #{lib}/pkgconfig/freetype2.pc].each do |f|
+      inreplace f, prefix, opt_prefix
+    end
```

Comment:
No objections to the second format either. Presumed ILZ preferred the first given he listed it first; sometimes I think he enjoys confusing me, heh.

Sent from my iPhone

> On 9 Sep 2016, at 05:38, Zhiming Wang <notifications@github.com> wrote:
> 
> In Formula/freetype.rb:
> 
> > @@ -34,6 +34,10 @@ def install
> >      system "./configure", "--prefix=#{prefix}", "--without-harfbuzz"
> >      system "make"
> >      system "make", "install"
> > +
> > +    %W[#{bin}/freetype-config #{lib}/pkgconfig/freetype2.pc].each do |f|
> > +      inreplace f, prefix, opt_prefix
> > +    end
> I like the second format too 👍
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>